apps/payment/models.py

```diff
@@ -553,3 +580,106 @@ def _send_receipt(self, payment_date, subject, description, payment_id, items, t
 
     class Meta:
         default_permissions = ('add', 'change', 'delete')
+
+
+class FikenSale(models.Model):
+    stripe_key = models.CharField(
+        'Stripe key',
+        max_length=20,
+        choices=StripeKey.ALL_CHOICES,
+        default=StripeKey.ARRKOM,
+    )
+    transaction_type = models.CharField(
+        'Transaction Type',
+        max_length=20,
+        choices=TransactionType.ALL_CHOICES,
+    )
+    original_amount = models.FloatField()
+    kind = models.CharField(
+        'Fiken Sale kind',
+        max_length=20,
+        choices=FikenSaleKind.ALL_CHOICES,
+        default=FikenSaleKind.CASH_SALE,
+    )
+    paid = models.BooleanField(default=True)
+    created_date = models.DateTimeField(default=timezone.now)
+    description = models.CharField(max_length=200)
```

Comment:
Could this be a TextField? Or is there a reason for the 200chars max length attribute?

Reply:
The 200 char limit comes from the Fiken API. I thought it would be nice to have the limit on the model as well

Reply:
I agree.